openedx/core/djangoapps/edx_global_analytics/README.md

```diff
@@ -0,0 +1,88 @@
+# edX global analytics
+
+`EdX global analytics` is part of edx-platform as Django application. It has arisen out of [Open edX](https://open.edx.org)
+requirement to collect edx-platform installations statistics data from all over the world.
+
+So one at 24 hours this application fetches and sends edX installation statistics to [Acceptor](https://github.com/raccoongang/acceptor) developed by @raccoongang team.
+Acceptor is required to be able to collect, visualize and process this data. In general, it provides possibilities
+for analysing trends of platform usage and users engagement in e-learning process per country and globally around the world.
+It is usually carried out via graphs, world map and activity metrics.
+
+## The way sent data looks like on Acceptor
+
+![acceptor_world_map](https://user-images.githubusercontent.com/22666467/27970248-d015fd9c-6356-11e7-906c-8c052cd65b08.png)
+
+## Types of statistics sent, and dependence on statistics level
+
+Firstly, `statistic level` is a string constant in settings (`lms.env.json`) which regulates size of bunch platform sends.
```

Comment:
Please add statement, that this is optional and all will work on paranoid level w/o any changes to settings